openassessment/xblock/staff_area_mixin.py

```diff
@@ -213,6 +229,18 @@ def render_staff_grade_form(self, data, suffix=''):  # pylint: disable=W0613
                     )
                     # Add team info to context
                     submission_context['teams_enabled'] = self.teams_enabled
+                    if self.teams_enabled:
+                        user = self.get_real_user(anonymous_student_id)
+
+                        if not user:
+                            logger.error(u'{}: User lookuip for anonymous_user_id {} failed'.format(self.location, anonymous_user_id))
```

Comment:
Python 3.0 doesn't require 'u' prefix to make strings unicode. This is probably copy and paste, but still worth mentioning and removing that 'u' prefix.

Reply:
will do